example-model/src/main/resources/application.yaml

```diff
@@ -22,6 +22,10 @@ spring:
   main:
     allow-bean-definition-overriding: true
 
+# Since using eureka, let spring autoconfigure a port
+server:
+  port: 0
+
 eureka:
   instance:
     hostname: localhost
```

Comment:
Wont this cause issues running the example in K8s? Or do we not care about that just yet and this will be re-worked as part of the ticket to add the K8s smoke test?

Reply:
its not needed and can be removed, its just a failsafe to make sure that we dont get port conflicts just yet. it can be left in and refactored with doing k8s or removed now

Reply:
leave it in then